pipeline/release-build.yaml

```diff
@@ -5,78 +5,183 @@ trigger: none
+
+      - script: 'move app-release-unsigned.apk AccessibilityInsightsforAndroidService.apk'
+        displayName: (signed-apk) rename APK file to remove unsigned qualifier
+        workingDirectory: '$(build.artifactstagingdirectory)/signed-apk'
+
+      - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+        displayName: '(signed-apk) sign release APK with ESRP CodeSigning'
+        inputs:
+          ConnectedServiceName: 'Accessibility Insights for Android Service AAD APP Id'
+          FolderPath: '$(build.artifactstagingdirectory)/signed-apk'
+          Pattern: '*.apk'
+          signConfigType: inlineSignParams
+          inlineOperation: |
+            [
+                    {
+                        "KeyCode" : "CP-458288-Java",
+                        "OperationCode" : "AndroidSign",
+                        "Parameters" : {},
+                        "ToolName" : "sign",
+                        "ToolVersion" : "1.0"
+                    },
+                    {
+                        "KeyCode" : "CP-458288-Java", 
+                        "OperationCode" : "JavaVerify",
+                        "Parameters" : {},
+                        "ToolName" : "sign",
+                        "ToolVersion" : "1.0"
+                    }
+            ]
+
+      - task: PublishPipelineArtifact@1
+        displayName: (signed-apk) publish artifact
+        inputs:
+          artifactName: 'signed-apk'
+          targetPath: '$(build.artifactstagingdirectory)/signed-apk'
+
+      # terms artifact
+
+      - task: CopyFiles@2
+        displayName: (terms) copy LICENSE/NOTICE to artifact staging
+        inputs:
+          contents: |
+            LICENSE
+            NOTICE.txt
+          sourceFolder: '$(system.defaultWorkingDirectory)'
+          targetFolder: '$(build.artifactstagingdirectory)/terms'
+
+      - task: PublishPipelineArtifact@1
+        displayName: (terms) publish artifact
+        inputs:
+          artifactName: 'terms'
+          targetPath: '$(build.artifactstagingdirectory)/terms'
+
+      # github-release-archive artifact
+
+      - task: CopyFiles@2
+        displayName: (github-release-archive) copy files to artifact staging
+        inputs:
+          contents: |
+            terms/LICENSE
+            terms/NOTICE.txt
+            signed-apk/AccessibilityInsightsforAndroidService.apk
+          sourceFolder: '$(build.artifactstagingdirectory)'
+          targetFolder: '$(build.artifactstagingdirectory)/github-release-archive/contents'
+          flattenFolders: true
+
+      - task: ArchiveFiles@2
+        displayName: '(github-release-archive) create release archive'
+        inputs:
+          rootFolderOrFile: '$(build.artifactstagingdirectory)/github-release-archive/contents'
+          includeRootFolder: false
+          archiveFile: '$(build.artifactstagingdirectory)/github-release-archive/AccessibilityInsightsForAndroid.zip'
```

Comment:
I feel like it would be more clear if this were suffixed with Service, but the existing releases and link from accessibilityinsights.io don't have Service. Maybe something for a future coordinated PR if we'd like?

Reply:
I agree that it would be more clear with the service suffix. That would require coordinating a change with the website though, like you mentioned, so let's leave it for a separate PR